dependencies.toml

```diff
@@ -0,0 +1,1032 @@
+[versions]
+akka = "2.6.19"
+bouncycastle = "1.70"
+brotli4j = "1.7.1"
+curator = "5.2.1"
+dagger = "2.42"
+dropwizard1 = "1.3.29"
+dropwizard2 = "2.1.0"
+errorprone = "2.14.0"
+eureka = "1.10.17"
+graphql-kotlin = "5.3.2"
+grpc-kotlin = "1.3.0"
+groovy = "3.0.5"
+guava = "31.1-jre"
+hamcrest = "2.2"
+jetty93 = '9.3.30.v20211001'
+jetty94 = "9.4.46.v20220331"
+jmh = "1.35"
+json-unit = "2.35.0"
+junit5 = "5.8.2"
+kotlin = "1.7.0"
+kotlin-coroutine = "1.6.3"
+micrometer = "1.9.1"
+micrometer13 = "1.3.20"
+mockito = "4.6.1"
+monix = "3.4.1"
+munit = "0.7.29"
+okhttp4 = "4.10.0"
+opensaml = "3.4.6"
+protobuf = "3.19.2"
+reactive-grpc = "1.2.3"
+reactive-streams = "1.0.4"
+reactor = "3.4.19"
+resteasy = "5.0.2.Final"
+retrofit2 = "2.9.0"
+sangria = "3.0.0"
+sangria-slowlog = "2.0.4"
+scala-java8-compat = "1.0.2"
+scalapb = "0.11.11"
+scalapb-json = "0.12.0"
+slf4j = "1.7.36"
+spring-boot1 = "1.5.22.RELEASE"
+spring-boot2 = "2.7.1"
+tomcat8 = "8.5.78"
+tomcat9 = "9.0.62"
+
+[boms]
+brave = { module = "io.zipkin.brave:brave-bom", version = "5.13.9" }
```

Comment:
So the convention of specifying version is that
- the version is inlined if it's used only once.
- the version is specified in `[versions]` section if it's used more than once.
Is that correct?

Reply:
Correct. But we can freely change the convention anytime. 😆